dev-python/mkdocs-bootswatch/mkdocs-bootswatch-1.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Bootswatch themes for MkDocs"
+HOMEPAGE="https://www.mkdocs.org https://github.com/mkdocs/mkdocs-bootswatch"
+SRC_URI="https://github.com/mkdocs/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="doc"
+
+DEPEND="dev-python/mkdocs"
```

Comment:
BDEPEND?

Reply:
Same as mkdocs-bootstrap, I put it in DEPEND because it is a theme for mkdocs